Calculator/Calculator/Model/LinkedList.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  CalculatorItemQueue.swift
+//  Calculator
+//
+//  Created by karen on 2023/06/07.
+//
+
+final class Node<Element> {
+    var data: Element
+    var next: Node?
+    
+    init(data: Element) {
+        self.data = data
+    }
+}
+
+struct LinkedList<Element> {
+    private var head: Node<Element>?
+    private var tail: Node<Element>?
+    
+    var isEmpty: Bool {
+        return head == nil
+    }
+    
+    var first: Element? {
+        return head?.data
+    }
+    
+    var last: Element? {
+        return tail?.data
+    }
+    
+    mutating func append(data: Element) {
+        let newNode = Node(data: data)
+        
+        if head == nil {
+            head = newNode
+            tail = newNode
+        } else {
+            tail?.next = newNode
+            tail = newNode
+        }
```

Comment:
좋네요! 같은 로직이 두번 반복되는거 같은데 다음과 같이 수정하는 건 어떨까요? 😁
```suggestion
        if head == nil {
            head = newNode
        } else {
            tail?.next = newNode
        }
        tail = newNode
```

Reply:
아하!! 그 부분을 또 미쳐 파악하지 못했네요😅
다음 스텝 진행할 때 수정해서 반영하도록 하겠습니다. 감사합니다 원비~☺️